master/reference/calicoctl/resources/ippool.md

```diff
@@ -29,27 +28,21 @@ spec:
 
 | Field       | Description                 | Accepted Values   | Schema |
 |-------------|-----------------------------|-------------------|--------|
-| cidr     | IP range to use for this pool.  | A valid IPv4 or IPv6 CIDR. | string |
+| name     |  The name of this IPPool resource. Required. | Alphanumeric string with optional `.`, `_`, `-`, or `/` | string |
```

Comment:
nit: extra space before "The name of this IPPool ..."

Reply:
Extra spaces get rendered correctly by jekyll